src/main/resources/view/RedTheme.css

```diff
@@ -0,0 +1,204 @@
+.background {
+    -fx-background-color: #ff4745;
+}
+
+.background2 {
+    -fx-background-color: #f4f4f4;
+}
+
+.text-field,.text-area, .split-pane,.tab {
+	-fx-border-color: #2e4053;
```

Comment:
Anyway if you want a style guide for CSS: https://oss-generic.github.io/process/codingStandards/CodingStandard-Css.html